packages/react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -29,6 +31,24 @@ const getClassNames = classNamesFunction<IPersonaCoinStyleProps, IPersonaCoinSty
   cacheSize: 100,
 });
 
+const getInitialsStyles = memoizeFunction(
+  (
+    className: string,
+    initialsBackgroundColor: PersonaInitialsColor | string | undefined,
+    initialsTextColor: string | undefined,
+    text: string | undefined,
+    primaryText: string | undefined,
+    showUnknownPersonaCoin: boolean | undefined,
+  ) =>
+    mergeStyles(
+      className,
+      !showUnknownPersonaCoin && {
+        backgroundColor: getPersonaInitialsColor(text, initialsBackgroundColor, primaryText),
```

Comment:
```suggestion
        backgroundColor: getPersonaInitialsColor({text, initialsBackgroundColor, primaryText}),
```
After you revert the changes for `getPersonaInitialsColor `, this ^ would work :)